junebug/logging_service.py

```diff
@@ -50,7 +50,7 @@ def logger_for_event(self, event):
         system = event.get('system', '-')
         parts = [self.worker_id]
         if system != '-':
-            parts.extend(system.split(','))
+            parts = system.split(',')
```

Comment:
No I guess we don't need that anymore. When `worker_id` is not in `system`, then it isn't logged. So the messages when `system` is `-` will mean that none of the channel workers will log it.